components/requests/RequestEditForm.tsx

```
@@ -1,48 +1,77 @@
+async function getGeoLocation(
+  address: string
+): Promise<{ lat: number; lon: number } | null> {
+  try {
+    const response = await fetch(
+      `https://nominatim.openstreetmap.org/search?format=json&q=${encodeURIComponent(
+        address
+      )}`
+    );
+    const data = await response.json();
+
+    if (data && data[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)